content/ar/docs/tutorials/_index.md

```diff
@@ -0,0 +1,59 @@
+---
+title: برامج تعليمية
+main_menu: true
+weight: 60
+content_type: concept
+---
+
+<!-- overview -->
+
+يحتوي هذا القسم من وثائق كوبرناتيز على برامج تعليمية.
+
+يُظهر البرنامج التعليمي كيفية تحقيق هدف يتجاوز مجرد [مهمة](/docs/tasks/). عادة يحتوي البرنامج التعليمي على أقسام مختلفة، ويحتوي كل قسم على خطوات مختلفة.
+
+قبل استكشاف كل درس، قد يكون من الفائدة الاحتفاظ بعلامة مرجعية للـ [المعجم الموحد](/docs/reference/glossary/) لتسهيل الرجوع إليه لاحقًا.
```

Comment:
I hope to check `[   ]` is correctly used. (to me it looks like `]     [`)

Reply:
We use it like this and it should be ok